lib/utils.h

```diff
@@ -78,4 +78,7 @@ inline static const char *getOrdinalText(int i)
     return "th";
 }
 
+template<typename T>
+inline void UNUSED(T&&) {}
```

Comment:
I would have implemented it as a macro.

I dont have a strong opinion about if macro or template is better.

As far as I see the assembler output is the same for (void)param and for the template. (g++ -O2).